_includes/try-geo.html

```diff
@@ -0,0 +1,25 @@
+<h1 class="ui horizontal divider"> Essayez moi </h1>
+<div class="ui aligned segment">
+    <div class="ui two column middle aligned very relaxed stackable grid">
+        <div class="column">
+            <div class="six wide column" id="mapid" style="height: 360px;"></div>
+        </div>
+        <div class="ui vertical divider"><i class="code icon"></i> </div>
+        <div class="center aligned column">
+          <div class="ui warning message">
+              <div class="header">Pas d'inquiétude<i class="warning icon"></i> </div>
+              <p>Le temps de chargement n'est pas représentatif des performances de l'api <i class="smile icon"></i></p>
+          </div>
+            <div id="commune" style="display:none;">
```

Comment:
Don't use inline styles.
If you need invisibility to be set in your HTML, use the `hidden` attribute.
Better: use a semantic class name, such as `no-js` if your intent is to hide stuff until JS is loaded.